kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/document/validation/configuration/RequisitionValidation.xml

```diff
@@ -50,5 +50,14 @@
       </list>
     </property>
   </bean>
+  

```

Comment:
You are correct. I tested without this code and everything worked as expected.  This has been reverted.